examples/03_Multitask_Exact_GPs/Hadamard_Multitask_GP_Regression.ipynb

```diff
@@ -58,8 +58,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "train_x1 = torch.rand(50)\n",
-    "train_x2 = torch.rand(50)\n",
+    "train_x1 = torch.rand(2000)\n",
```

Comment:
By how much does this increase runtime? Should we be concerned here if this is run as part of the unit tests?

Reply:
This change (plus the ones in the other two examples) add at most one second to the test suite - I just checked.